examples/custom_filter_example.py

```diff
@@ -0,0 +1,114 @@
+from typing import Any, Union
+from aiogram import Bot, Dispatcher, executor, types
+from aiogram.dispatcher.filters import BoundFilter
+
+API_TOKEN = "BOT_TOKEN_HERE"
+
+
+LIMIT_DATABASE_RECORDS = 6
+
+
+ADMIN_IDS = [
+    000000000,
+    111111111,
+    222222222,
+    333333333,
+    444444444,
+]
+
+
+bot = Bot(token=API_TOKEN)
+dp = Dispatcher(bot)
+
+
+class GlobalAdminFilter(BoundFilter):
+    """
+    Check if user bot admin
+    """
+
+    key = "global_admin"
+
+    def __init__(self, global_admin: bool):
+        self.global_admin = global_admin
+
+    async def check(self, obj: Union[types.Message, types.CallbackQuery]):
+        user = obj.from_user
+        if user.id in ADMIN_IDS:
+            return self.global_admin is True
+        return self.global_admin is False
+
+
+class IsEvenIdFilter(BoundFilter):
+    """
+    Check if user.id is even
+    """
+
+    key = "is_even"
+
+    def __init__(self, is_even: int):
+        if isinstance(is_even, int):
+            self.is_even = is_even
+        else:
+            raise ValueError(
+                f"filter is_even must be a int, not {type(is_even).__name__}"
+            )
+
+    async def check(self, obj: Union[types.Message, types.CallbackQuery]):
+        user_id = obj.from_user.id
+        return bool(user_id % self.is_even == 0)
+
+
+class DatabaseIsNotEmptyFilter(BoundFilter):
+    """
+    Check if the bot admins users contains the required number of records
+    """
```

Comment:
Inconsistent filter name and docstring.
Also, the filter always returns a positive result.